andersen-homework/src/main/java/org/model/Ticket.java

```diff
@@ -0,0 +1,51 @@
+package org.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Objects;
+
+
+@Getter
+@Setter
+
+public class Ticket {
+
+    public String startDate;
```

Comment:
why public modifier here?

Reply:
when i selected LocalDate as a modifier , I get this error in validate method Cannot resolve method 'isEmpty' in 'LocalDate' therefore I used string as a modifier.

Reply:
i did not do it intentionally, so I ll change it to private

Reply:
because when i chose private, i get this error in validate method 'ticketType' has private access in 'org.model.Ticket'

Reply:
now i have solved it with get method

Reply:
Don't see the result in the pr

Reply:
check it again please, teacher